aws-integration/aws-otel-collector/values.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: otel-collector
+  namespace: ${KYMA_NS}
```

Comment:
Consuming the namespace from the environment will not work. Maybe remove the namespace from the yaml files and add them to your `kubectl` calls.